software/src/algorithms/navigation.test.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * navigation.test.c
+ */ 
+
+
+#include <testing.h>
+#include <error.h>
+
+#include "navigation.h"
+
+TEST(navigation compiles, src/algorithms/navigation.c, src/error.c) {
```

Comment:
You need to have a comma between each dependent file.  Earlier you had just a space in here.